### DISCUSSION TITLE
Add ESP32 tick boxes for Matter and LVGL

### DISCUSSION THREAD

**Discussion by Jason2866** [benzino77/tasmocompiler#398] [Ideas]
Hi @benzino77 

the mentioned features are missing and used from many Tasmota users ;-)

**Comment by benzino77** [comment]
Will do ;)
... after my vacations.

**Reply by benzino77** [reply]
Ok. So Matter protocol is already there (development branch).

**Reply by benzino77** [reply]
@Jason2866 correct me if I'm wrong
1. "regular" display support is available for esp8266 and esp32 boards
2. LVGL is dedicated for esp32 boards
3. "regular" display and LVGL are exclusive

**Comment by Jason2866** [comment]
1) yes with uDisplay driver 
2) yes
3) what do you mean with regular and exclusive?

uDisplay drivers can be used with esp8266 and esp32x. LVGL and Haspmota do need an esp32x chip. The implementation of LVGL and Haspmota is done with solidified Berry code. Berry is only possible with esp32x MCUs

**Reply by benzino77** [reply]
That is my fault. I should wrote: "mutually exclusive" instead of just "exclusive"
What I meant by that is: user can choose (from TasmoCompiler GUI perspective) "old/regular" support for displays OR lvgl - not both (different set of `#define` and `#undef`)

**Comment by Jason2866** [comment]
LVGL and Haspmota does include the possibility to use Displaytext commands too.

**Reply by benzino77** [reply]
Thanks. The `development` version of TasmoCompiler is there so feel free to check its new functionality.

**Reply by benzino77** [reply]
New version was released.

**Comment by Jason2866** [comment]
Thx!
